allennlp/models/encoder_decoders/wikitables_semantic_parser.py

```diff
@@ -1063,6 +1069,9 @@ def __init__(self,
         # TODO(pradeep): Do not hardcode decoder cell type.
         self._decoder_cell = LSTMCell(input_dim, output_dim)
 
+        check_dimensions_match(output_dim, mixture_feedforward.get_input_dim(),
```

Comment:
You should probably also check that the output dimension of the mixture feedforward is 1.

Reply:
done.